Makefile

```diff
@@ -429,6 +429,34 @@ build-archive:
 release-unix: clean full build-archive
 	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
 
+# Set MacOS/Darwin cert vars for signing/notarizing
+# TSH_SKELETON is a directory name relative to build.assets/macos/
+ENVIRONMENT_NAME ?= promote
+
+# TODO(camh): Verify key names for production keys
+DEVELOPER_KEY_NAME_promote = Developer ID Application: Ada Lin
```

Comment:
Is "Ada Lin" expected here and below?

Reply:
Also, dding comments on all these variables to explain what they are for would be helpful for future maintainers.

Reply:
The production promote/release certificate info can be found here:

https://goteleport.com/security/

It sounds like we're ready to have those secrets configured in https://github.com/gravitational/teleport.e/issues/832, right @camscale?

Reply:
dding? 

I'm not sure what the names for the promote keys are yet. @wadells Do you have these set up in the `promote` environment yet? Can you let me know?

Reply:
Yep, ready for the `promote` environment now/shortly. This all works with the `build` environment. I expect I'll merge with `build` and land a couple more PRs on top, and then switch to promote and ensure the end-to-end release process works with that.